app/cdap/components/ConnectionList/index.tsx

```diff
@@ -198,76 +280,138 @@ export default function ConnectionList() {
-                    </Typography>
+                        <SearchRoundedIcon />
+                        <input
+                          type="text"
+                          className={classes.searchBar}
+                          onChange={(e) => handleSearch(e, index)}
+                          ref={(e) => {
+                            refs.current[index] = e;
+                          }}
+                        />
+                        <Box
+                          className={classes.closeIcon}
+                          onClick={(e) => handleClearSearch(e, index)}
+                        >
+                          <CloseIcon />
+                        </Box>
+                      </Box>
+                    </>
+                  );
+                }
+                return (
+                  <Box className={classes.tabsContainerWithHeader}>
+                    <Box className={classes.tabHeaders}>{headerContent}</Box>
+                    <ConnectionsTabs
+                      tabsData={eachFilteredData}
+                      handleChange={selectedTabValueHandler}
+                      value={eachFilteredData.selectedTab}
+                      index={index}
+                      connectionId={connectionId || ''}
+                      toggleLoader={(value: boolean, isError?: boolean) =>
+                        toggleLoader(value, isError)
+                      }
+                      setIsErrorOnNoWorkSpace={setIsErrorOnNoWorkSpace}
+                    />
                   </Box>
                 );
-            }
-            return (
-              <Box className={classes.tabsContainerWithHeader}>
-                <Box className={classes.tabHeaders}>{headerContent}</Box>
-                <ConnectionsTabs
-                  tabsData={each}
-                  handleChange={selectedTabValueHandler}
-                  value={each.selectedTab}
-                  index={index}
-                  connectionId={connectionId || ''}
-                  toggleLoader={(value: boolean, isError?: boolean) => toggleLoader(value, isError)}
-                  setIsErrorOnNoWorkSpace={setIsErrorOnNoWorkSpace}
-                />
+              })}
+          </SelectDatasetWrapper>
+          {tabSize < 4 && (
+            <Box className={classes.infographContainer}>
+              <Box className={classes.infograph}>
+                <InfoGraph />
               </Box>
-            );
-          })}
-        </SelectDatasetWrapper>
+            </Box>
+          )}
+        </Box>
       ) : (
-        <Typography className={classes.noDataLabel}>
-          {' '}
-          {T.translate('features.NewWranglerUI.ConnectionsList.labels.noConnections')}
-        </Typography>
-        // TODO: No connectors types are available screen needs to be appended here
+        <>
+          {loading && (
+            <NoConnectionsScreen
+              title={T.translate('features.NewWranglerUI.NoRecordScreen.connectionsList.title')}
```

Comment:
use PREFIX variable
write it in constnts.ts
and append the specific path to it

Reply:
Done